TrackletGraph.py

```diff
@@ -330,7 +333,46 @@ def get_mem_dict_from_config(fname_mconfig, region='-'):
             elif region == 'D': # disk project
                 if not isdisk or isbarrel:
                     continue
-            
+
```

Comment:
@meisonlikesicecream As this code touches memories written by VMR, please check it's compatible with your IR-VMR PR.